pkg/config/config.go

```diff
@@ -144,6 +153,16 @@ type Sources struct {
 	DefaultMessage string `toml:"default_message"`
 }
 
+// Client are the config options for the client.
+type Client struct {
+	KeycloakURL      string `toml:"keycloak_url" json:"keycloak_url"`
+	KeycloakRealm    string `toml:"keycloak_realm" json:"keycloak_realm"`
+	KeycloakClientID string `toml:"keycloak_client_id" json:"keycloak_client_id"`
+	UpdateInterval   int    `toml:"update_interval" json:"update_interval"`
```

Comment:
Okay, it's for the client,  but it feels a bit off as we use time.Duration for time intervals used by the server.

Reply:
Done